cms/djangoapps/contentstore/views/course.py

```diff
@@ -458,12 +458,14 @@ def filter_ccx(course_access):
     return courses_list.values(), in_process_course_actions
 
 
-def _accessible_libraries_iter(user):
+def _accessible_libraries_iter(user, org=None):
     """
-    List all libraries available to the logged in user by iterating through all libraries
+    List all libraries available to the logged in user by iterating through all libraries.
+
+    If 'org' is present, only libraries from that org will be returned.
     """
     # No need to worry about ErrorDescriptors - split's get_libraries() never returns them.
-    return (lib for lib in modulestore().get_libraries() if has_studio_read_access(user, lib.location.library_key))
+    return (lib for lib in modulestore().get_libraries(org=org) if has_studio_read_access(user, lib.location.library_key))
```

Comment:
Yeah, we'd just need to change this line I think - but I'm not 100% confident that nothing relies on the current behavior either.

https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/modulestore/split_mongo/split.py#L539